### PR TITLE
Add `geo.name` as the field for ad hoc location names.

### PR DESCRIPTION
**Pull request by webmat**
This is useful for representing location manually, in some situations.

This idea originated in https://github.com/elastic/beats/pull/9392

### CODE REVIEW

**Comment by webmat** [comment]
Tentatively tagging for beta 2. I love this concept!

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ All notable changes to this project will be documented in this file based on the
 * Add `user.full_name` field. #201
 * Add `network.community_id` field. #208
 * Add fields `geo.country_name` and `geo.region_iso_code`. #214
+* Add \`geo.name\` for ad hoc location names. #248
```
Comment (ruflin) [inline CHANGELOG.md]: Nit: Are the `\` on purpose?

Comment (webmat) [reply CHANGELOG.md]: Oh no. That's a habit from code quoting field names in commit messages. Will remove ;-)

**Comment by andrewvc** [comment]
What, this is awesome! Much appreciated @webmat !

**Pull request merged**